packages/data-access/specs/request-block-specification-v0.1.0.md

```diff
@@ -0,0 +1,80 @@
+# Request Block Specification v0.1.0
+
+You can be interested in this document if:
+
+- You want to create your own implementation of the request protocol
+- You are curious enough to dive and see what is under the hood of the request protocol
+
+You don't need to read this document if:
+
+- You want to develop an app using the request protocol (see the API library instead [here](/packages/request-client.js))
+
+The Data Access layer is responsible for:
+
+- Indexing transactions to allow retrieval
+- Batching transactions
```

Comment:
data-access is also responsible to synchronize with storage and manage the local cache

Reply:
This is not in the scope of the request block

Reply:
it can be clearer with an introduction explaining what this document is about